daiquiri/core/management/commands/find_users.py

```diff
@@ -0,0 +1,142 @@
+import csv
+import re
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from allauth.account.models import EmailAddress
+from daiquiri.auth.models import Profile
+
+# from rdmo.projects.models import Membership
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument("--id", default=".*", help="find users by id")
+        parser.add_argument("--email", default=".*", help="find users by email")
+        parser.add_argument("--username", default=".*", help="find users by username")
+        parser.add_argument(
+            "--first_name", default=".*", help="find users by first name"
+        )
+        parser.add_argument("--last_name", default=".*", help="find users by last name")
+        parser.add_argument(
+            "--email_verified",
+            default=".*",
+            help="filter users by email_verified, requires argument, supports: true/false/1/0",
+        )
+        parser.add_argument(
+            "--profile_pending",
+            default=".*",
+            help="filter users by profile_pending, requires argument, supports: true/false/1/0",
+        )
+        parser.add_argument(
+            "--profile_confirmed",
+            default=".*",
+            help="filter users by profile_confirmed, requires argument, supports: true/false/1/0",
+        )
+        parser.add_argument(
+            "-p",
+            "--print",
+            action="store_true",
+            help="print found users, don't save them to csv",
+        )
+        parser.add_argument(
+            "-o",
+            "--output_file",
+            default="found_users.csv",
+            help="output file, default is 'found_users.csv'",
+        )
+
+    def save_csv(self, data, filename):
+        if len(data) > 0:
+            data_file = open(filename, "w")
+            csv_writer = csv.writer(data_file)
+            csv_writer.writerow(list(data[0].keys()))
+            for user in data:
+                csv_writer.writerow(user.values())
+            print("List written to " + filename)
+
+    def print_file(self, filename):
+        f = open(filename, "r")
+        content = f.read()
+        print(content)
+        f.close()
+
+    def get_profile(self, username):
+        return Profile.objects.get(user__username=username)
+
+    def rx_match(self, regex, s):
+        return bool(re.search(regex, str(s)))
+
+    def make_bool(self, val):
+        if str(val) == "1":
+            return True
+        if str(val).lower() == "true":
+            return True
+        return False
+
+    def bools_are_equal(self, b1, b2):
+        return self.make_bool(b1) == self.make_bool(b2)
+
+    def check_match(self, user, profile, email_verified, options):
+        if self.rx_match(options["id"], user.id) is False:
```

Comment:
again here I'd suggest `options.get("id", None)` but that is up to you. `options.get("id")` also works.